presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -107,7 +107,7 @@ public boolean schemaExists(JdbcIdentity identity, String schema)
             return tableHandle.getColumns().get();
```

Comment:
First of all we need to fix `getColumns`. Then we need to use same approach for other methods that are using cache here.

Session contains properties that may influence the behavior of the method. We need to use them in the cache key. Please handle that in separate commit.

Reply:
I'm not sure but most probably this comment is outdated and implemented.